.github/workflows/pr-handling.yaml

```diff
@@ -12,16 +12,16 @@ name: Assign PR to author and reviewers
 
 "on":
   pull_request_target:
-    types: [ opened, reopened, ready_for_review ]
+    types: [ labeled, unlabeled, opened, reopened, edited, ready_for_review, synchronize ]
 
 jobs:
   assign-pr:
     name: Assign PR to author
-    permissions:
-      contents: write
     runs-on: ubuntu-latest
     steps:
       - uses: toshimaru/auto-author-assign@v2.1.0
+        with:
+          repo-token: ${{ secrets.ORG_AUTOMATION_TOKEN }}
```

Comment:
Now this job will execute also when the PR is labeled and unlabeled as well as edited and synchronized. Don't we need to have checks to avoid running this unnecessarily?